projects/metis/src/app/dataset/lastexecution/lastexecution.component.ts

```diff
@@ -48,8 +49,19 @@ export class LastExecutionComponent {
     this.setReportMsg.emit(req);
   }
 
+  /** getPluginHighlighted
+  /* template utility to determine if row should be highlighted
+  /* @param { PluginExecution } plugin - the plugin to evaluate
+  /* @return boolean
+  */
+  getPluginHighlighted(plugin: PluginExecution): boolean {
+    return plugin.pluginStatus === PluginStatus.RUNNING;
+  }
```

Comment:
I think, as a result from an earlier comment, this method would no longer be necessary.

Reply:
I'll move it to a setter for the plugin `Input`